pkg/controllers/subnetport/subnetport_controller.go

```diff
@@ -781,8 +781,8 @@ func (r *SubnetPortReconciler) CheckAndGetSubnetPathForSubnetPort(ctx context.Co
 	return
 }
 
-func (r *SubnetPortReconciler) updateSubnetStatusOnSubnetPort(subnetPort *v1alpha1.SubnetPort, nsxSubnet *model.VpcSubnet) error {
-	gateway, prefix, err := r.SubnetPortService.GetGatewayPrefixForSubnetPort(nsxSubnet)
+func (r *SubnetPortReconciler) updateSubnetStatusOnSubnetPort(subnetPort *v1alpha1.SubnetPort, nsxSubnet *model.VpcSubnet, subnetCR *v1alpha1.Subnet) error {
```

Comment:
Also remove subnetCR in this function parameter?

Reply:
Done, thanks!